data/hails.txt

```diff
@@ -2030,7 +2030,273 @@ phrase "friendly author"
 
 phrase "friendly disabled"
 	word
-		"Our ship has been disabled! Please come board our ship and patch us up!"
+		"Our ship has been disabled! "
+		"We've gotten ourselves in a bit of a situation! "
+		"We've lost control of our ship! "
+		"Our ship took quite the pounding! "
+		"Hey! "
+		"S.O.S! S.O.S! "
+	word
+		"Please come board our ship and patch us up!"
+		"We would appreciate any help getting us running again!"
+		"We can't fix this on our own!"
+		"Could you come over and help us please?"
+		"Requesting emergency repairs!"
+		
+phrase "friendly disabled"
+	word
+		"Hey! "
+		""
+		""
+		"You're a friendly, right? "
+		"You answered! "
+		"Good to see a friendly face! "
+		""
+		"Hey, pleaase! "
```

Comment:
double a in please
and why not put all "" (empty words) after each other instead of spread out?

Reply:
Because we have three `""`s here already. If these spaces were moved into its own separate word, we could end up with
`" Our ship is disabled, could you lend a hand?"` with an extra space at the front.